src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -397,7 +397,7 @@ public Build getLastBuild() {
     }
 
     public void disable() {
-        check("disable");
+        check("Disable this project");
```

Comment:
Hum... Does it change the semantic?

Reply:
it's the label, passes all tests here that use it (quite a few)

I've adjusted javadoc in https://github.com/jenkinsci/acceptance-test-harness/pull/717/commits/57808b234e29b7bdb8f842d611d5a4c6e8b7b05a